pkg/controller/migmigration/annotation.go

```diff
@@ -103,6 +109,14 @@ func (t *Task) annotatePVCs(client k8sclient.Client, pod corev1.Pod) ([]string,
 		// Add to Restic volume list.
 		if pvAction == migapi.PvCopyAction {
 			volumes = append(volumes, pv.Name)
+			// PV storageClass annotation needed by the velero plugin.
+			storageClass := findPVStorageClass(pvs, pvc.Spec.VolumeName)
+			pvc.Annotations[PvStorageClassAnnotation] = storageClass
```

Comment:
Is it intentional to set the annotation when `storageClass` is an empty string ? unlike `accessMode` annotation. @sseago

Reply:
@pranavgaikwad It is, because we should never get here if it's empty for storageClass, since an empty storageClass results in an error condition on the MigPlan. For accessMode, empty is the default, as we only set it if the user intends to override the accessMode of the src PVC. We could still check for it being empty, but I don't know if we'll ever actually get here in that case.